Gemfile

```diff
@@ -4,5 +4,8 @@ group :test do
   gem 'rake'
   gem 'rspec'
   gem 'beaker', '~>1.17.0'
+  if ENV['GEM_SOURCE']
```

Comment:
`GEM_SOURCE` is the environment variable we use to point bundler to `rubygems.delivery.puppetlabs.net` in CI.